spec/controllers/articles_controller_spec.rb

```diff
@@ -122,7 +122,12 @@
         article1 = create(:article)
         article2 = create(:article)
         get :index
-        expect(assigns(:articles)).to eq([article2, article1])
+
+        assigned_articles = assigns(:articles)
+        binding.pry
+        assigned_articles_dates = assigned_articles[0]['created_at']
```

Comment:
you can remove this line see comment below

Reply:
I'd prefer to leave it as it is as I find it more readable, thank you for the suggestion though.